src/Details/Sections/General/Providers.js

```diff
@@ -9,16 +9,24 @@ const FIELD_NAME = 'providerName';
 
 export const Providers = props => {
   const { formatMessage } = useIntl();
-  const field = useField(FIELD_NAME);
+  const { options, isLoading } = useProvidersOptions();
+
+  const field = useField(FIELD_NAME, {
+    validate: value => !(options?.some(option => option.value === value)) &&
+      formatMessage({ id: 'ui-remote-storage.details.providerName.error' }),
+  });
 
-  const options = useProvidersOptions();
   const label = formatMessage({ id: 'ui-remote-storage.details.providerName' });
+  const dataOptions = !isLoading ? options : [{ label: formatMessage({ id: 'ui-remote-storage.loading' }) }];
+  const placeholder = isLoading ? undefined : formatMessage({ id: 'ui-remote-storage.select' })
```

Comment:
do you have such requirements to show loading placeholder? unusual pattern

Reply:
no specific requirements, no
just common sense